### PR TITLE
Initial stab at some governance type of docs

### PR DESCRIPTION
**Pull request by duglin**
First pass - I think its more important to get something in there that's a step in the right direction, and then tweak with follow-on PRs to make it more right.

Signed-off-by: Doug Davis <dug@us.ibm.com>

---
This change is [Reviewable](https://reviewable.io/reviews/istio/istio/5)

### CODE REVIEW

**Comment by louiscryan** [comment]
Review status: 0 of 6 files reviewed at latest revision, 1 unresolved discussion.

---

*[REVIEWING.md, line 15 at r1](https://reviewable.io:443/reviews/istio/istio/5#-KbCLPl3nNnUWI8PNVGY:-KbCLPl3nNnUWI8PNVGZ:b6vvbcb) ([raw file](https://github.com/istio/istio/blob/c501337109d4ec0ecc95a5e4826dd812b03aaf2c/REVIEWING.md#L15)):*
> ```Markdown
> to the project. Active contributors to the project will eventually get noticed
> by the maintainers, and one of the existing maintainers will nominate the
> contributor to become a maintainer.  A 'yes' vote from a majority (>50%) of
> ```

Bar seems a bit low, ideally would be 100%

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/istio/istio/5)*

**Comment by louiscryan** [comment]
Largely LGTM, one nit about voting in maintainers

---







*Comments from [Reviewable](https://reviewable.io:443/reviews/istio/istio/5#-:-KbCLxQHMatzXDEjJF8J:b-kyvcjk)*

**Comment by duglin** [comment]
I couldn't figure out how to reply to @louiscryan's comment :-)  so,
> Bar seems a bit low, ideally would be 100%

I didn't want to be held up by people who are off doing other things, on vacation, or just slow to respond.  Would 75% be ok?

**Comment by louiscryan** [comment]
75% sounds dandy

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,34 +1,86 @@
 # Contributing guidelines
 
-So, you want to hack on Istio? Yay!
```
Comment (smawson) [inline CONTRIBUTING.md]: Can we leave in some of the "fun"?  :)

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,34 +1,86 @@
...
+The following sections outline the process by which all changes to the Istio
```
Comment (smawson) [inline CONTRIBUTING.md]: remove "by"

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,34 +1,86 @@
...
+repositories will go through.  All changes, regardless of whether they are from
+newcomers to the community or from the core team itself follow the exact
```
Comment (smawson) [inline CONTRIBUTING.md]: drop "itself"

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -1,34 +1,86 @@
...
+- [Contributing a feature](#contributing-a-feature)
+- [Pull Requests](#pull-requests)
 
-## Contributor license agreements
+## Contributor License Agreements
 
-We'd love to accept your patches! Before we can take them, you have to jump a
-few legal hurdles.
+We'd love to accept your patches! But, before we can take them, you have 
+fill out the [Google CLA](https://cla.developers.google.com).
```
Comment (smawson) [inline CONTRIBUTING.md]: But before we can take them, you will have to fill out the ...

File: DEVGUIDE.md [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+
+The Istio Manager is used to configure Istio and propagate configuration to 
+the other components of the system, including the Istio mixer and the Istio 
+proxy mesh.
+
+### [mixer](https://github.com/istio/mixer)
+
+The Istio mixer provides the foundation of the Istio service mesh design. 
+It is responsible for insulating the Istio proxy and Istio-based services 
+from details of the current execution environment, as well as to implement
```
Comment (smawson) [inline DEVGUIDE.md]: , as well as implementing the control policies ...

File: DEVGUIDE.md [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+### [mixer](https://github.com/istio/mixer)
+
+The Istio mixer provides the foundation of the Istio service mesh design. 
+It is responsible for insulating the Istio proxy and Istio-based services 
+from details of the current execution environment, as well as to implement 
+the control policies that Istio supports.
+
+### [mixerclient](https://github.com/istio/mixerclient)
+
+C++ client for the mixer API.
```
Comment (smawson) [inline DEVGUIDE.md]: Will probably have other languages over time, so maybe just say client libraries for the mixer?

File: OWNERS [hunk 7]
```diff
@@ -0,0 +1,2 @@
+# This file contains the current list of maintainers of the project
+# Format: First Last <email>  - in alphabetical order by first name
```
Comment (smawson) [inline OWNERS]: Should we add the list of maintainers or doing that in a followup?

File: CONTRIBUTING.md (hunk 1 above)
Comment (duglin) [inline CONTRIBUTING.md]: fun? this is work/serious business!  :-)
Added it back in.

File: CONTRIBUTING.md (hunk 3 above)
Comment (duglin) [inline CONTRIBUTING.md]: done

File: CONTRIBUTING.md (hunk 4 above)
Comment (duglin) [inline CONTRIBUTING.md]: done

File: DEVGUIDE.md (hunk 5 above)
Comment (duglin) [inline DEVGUIDE.md]: done

File: DEVGUIDE.md (hunk 6 above)
Comment (duglin) [inline DEVGUIDE.md]: done

File: OWNERS (hunk 7 above)
Comment (duglin) [inline OWNERS]: yea, I'd like to but I don't know all of the github IDs we want added.

**Comment by duglin** [comment]
I think I got all of the current comments addressed.

**Comment by smawson** [comment]
I think you need to push the new version of the files?  I don't see any changes since my last comments.

**Comment by duglin** [comment]
oops sorry - thought I did that before I closed my laptop lid, should be there now

**Comment by smawson** [comment]
Sorry for delay, let's submit and any changes people can just send a PR.

**Pull request merged**